content/docs/other-guides/authentication.md

```diff
@@ -0,0 +1,514 @@
++++
+title = "Kubeflow Authentication with Istio and Dex"
+description = "Instructions for installing Kubeflow with authentication support"
+weight = 4
++++
+
+This installation of Kubeflow is maintained by [Arrikto](https://www.arrikto.com/), is geared towards existing Kubernetes clusters and does not depend on any cloud-specific feature.
+
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
+
+![platform existing architecture](https://i.imgur.com/OlaN73j.png)
+
+### Prerequisites
+- Kubernetes Cluster with LoadBalancer support.
```

Comment:
@yanniszark @johnugeorge Can we link to K8s docs describing this?

Reply:
done